WordPress/gutenberg
# Changelog tool: update Automated Testing label

## What?
Updates the label type mapping to match the one being used currently so that PRs labeled `[Type] Automated Testing` are correctly categorized under Tools

## Testing Instructions
- Run `npm run other:changelog -- --milestone="Gutenberg 16.6"` — PRs such as [this one](https://github.com/WordPress/gutenberg/pull/53856) should be correctly categorized under Tools.
